src/Controller/RunController.php

```diff
@@ -271,9 +271,11 @@ public function compare(Request $request): void
             'base_run' => $baseRun,
             'head_run' => $headRun,
             'candidates' => $candidates,
-            'url_params' => $request->get(),
+            // 'url_params' => $request->get(),
```

Comment:
why is this commented out? remove dead code.

Reply:
ok i will re-pr

Reply:
what? just add more commits!

Reply:
sry, im wrong expression, you can check this pr, already 2 commits.